src/test/java/plugins/SubversionPluginTest.java

```diff
@@ -49,7 +54,8 @@ public void run_basic_subversion_build() throws SubversionPluginTestException {
         f.addShellStep("test -d .svn");
         f.save();
 
-        f.startBuild().shouldSucceed().shouldContainsConsoleOutput("test -d .svn");
+        Build b = f.startBuild().shouldSucceed();
+        assertThat(b.getConsole(), Matchers.containsRegexp("test -d .svn", Pattern.MULTILINE));
```

Comment:
I'm confused, sorry! Is it a +1 when I change it to use the <code>org.hamcrest.Matchers.containsString</code> or you find it ok already?